site/en/install/gpu.md

```diff
@@ -98,15 +98,15 @@ complicates installation of the NVIDIA driver and is beyond the scope of these i
 # Install development and runtime libraries (~4GB)
 <code class="devsite-terminal">sudo apt-get install --no-install-recommends \
     cuda-10-0 \
-    libcudnn7=7.4.1.5-1+cuda10.0  \
-    libcudnn7-dev=7.4.1.5-1+cuda10.0
+    libcudnn7=7.6.0.64-1+cuda10.1  \
+    libcudnn7-dev=7.6.0.64-1+cuda10.1
```

Comment:
the `cuda10.1` version of the cuDNN library is no problem - everything else will stay cuda 10.0 - also no dependency problems

Reply:
``` 
sudo apt-get install --no-install-recommends \
        cuda-10-0 \
        libcudnn7=7.6.0.64-1+cuda10.0  \
        libcudnn7-dev=7.4.1.5-1+cuda10.0
```
works too its the same file